STARTERKIT/primary/src/css/styles.css

```diff
@@ -0,0 +1,22 @@
+@import "../../node_modules/tailwindcss/base.css";
```

Comment:
should be just 
`@import "tailwindcss/base.css";`

Reply:
also i'm not sure we need that normalize form base.css. in this case maybe remove core one?

Reply:
https://tailwindcss.com/docs/preflight#disabling-preflight